### DISCUSSION TITLE
创建资源错误

### DISCUSSION THREAD

**Discussion by fujunit** [apache/dolphinscheduler#7398] [Q&A]
使用ambari的插件搭建dolphinscheduler之后，创建资源报错
org.apahce.dolphinscheduler.api.exception.ApiExceptionHandler:[45]-创建资源错误
java,lang.NullPointerException:Name is null

**Comment by fujunit** [comment]
用的版本是1.3.0和1.3.3的，创建租户、用户、资源都会报这个空指针错误

**Comment by davidzollo** [comment]
please paste the full stack trace info
